integration/utk_test.go

```diff
@@ -94,7 +97,19 @@ func TestExtractAssembleExtract(t *testing.T) {
 					t.Errorf("no files in directory %q", d)
 				}
 			}
-
+			sedRemove := func(path string) {
+				sedCmd := exec.Command("sed", "-i", "/\"ExtendedSize\": [0-9]*.*/d", path)
```

Comment:
we can't have dependencies on sed in a test, can you change how this is done.

Reply:
what you can do, instead, is use regexp.ReplaceAll; just read the file in, do the replace, write it out.
os.ReadAll
regexp.ReplaceAll
os.WriteFile
but what I'm wondering: if you don't want ExtendedSize in the test, why did you put it in the JSON? If it's not in the JSON, then the struct member will be zero; is that not good enough?